src/views/Admin/Index.vue

```diff
@@ -26,6 +26,17 @@ export default {
     },
   },
 
+  watch: {
+    'namespace.name': {
+      immediate: true,
+      handler (name) {
+        document.title = name
```

Comment:
Why not just watch $route.to and set it to `${this.$t('general:label.app-name')} - Admin`
Also are we ignoring the non translated - Admin part?